omnisafe/adapter/modelbased_adapter.py

```diff
@@ -106,30 +113,27 @@ def __init__(  # pylint: disable=too-many-arguments
         else:
             self.task = None
 
-        self._ep_ret: torch.Tensor
-        self._ep_cost: torch.Tensor
-        self._ep_len: torch.Tensor
         self._current_obs, _ = self.reset()
-        self._max_ep_len = 1000
+        self._max_ep_len: int = 1000
         self._reset_log()
-        self._last_dynamics_update = 0
-        self._last_policy_update = 0
-        self._last_eval = 0
-        self._first_log = False
+        self._last_dynamics_update: int = 0
+        self._last_policy_update: int = 0
+        self._last_eval: int = 0
+        self._first_log: bool = False
 
-    def get_cost_from_obs_tensor(self, obs: torch.Tensor) -> torch.Tensor | None:
+    def get_cost_from_obs_tensor(self, obs: torch.Tensor) -> torch.Tensor:
         """Get cost from tensor observation.
 
         Args:
-            obs (torch.Tensor): The observation.
+            obs (torch.Tensor): The tensor version of observation.
         """
         return (
             self._env.get_cost_from_obs_tensor(obs)
             if hasattr(self._env, 'get_cost_from_obs_tensor')
-            else None
+            else torch.zeros(1)
         )
 
-    def get_lidar_from_coordinate(self, obs: torch.Tensor) -> torch.Tensor | None:
+    def get_lidar_from_coordinate(self, obs: np.ndarray) -> torch.Tensor | None:
```

Comment:
Original version `None` is pretty bad because there is no `if` `else` process to deal with `None`, here use zero tensor as default.